eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -183,3 +183,21 @@ jobs:
         ServiceDirectory: ${{ parameters.ServiceDirectory }}
         BuildTargetingString: ${{ parameters.BuildTargetingString }}
         TestMarkArgument: ${{ parameters.TestMarkArgument }}
+
+  - job: 'RunRegression'
+    condition: and(succeededOrFailed(), eq(variables['Run.Regression'], 'true'))
+    displayName: 'Run Regression'
+    variables:
+    - template: ../variables/globals.yml
+
+    dependsOn:
+      - 'Build'
+
+    pool:
+      vmImage: 'windows-2019'
```

Comment:
Nit: Making the default run on windows will make this thing run a lot slower. Recommend linux.